mne_qt_browser/_pg_figure.py

```diff
@@ -1809,19 +1815,22 @@ def __init__(self, main, title="Settings", **kwargs):
         ch_scaling_layout = QFormLayout()
         self.ch_scaling_spinboxes = {}
 
-        # Get all unique channel types
+        # Get all unique channel types and allow scaling
         ordered_types = self.mne.ch_types[self.mne.ch_order]
         unique_type_idxs = np.unique(ordered_types, return_index=True)[1]
         ch_types_ordered = [ordered_types[idx] for idx in sorted(unique_type_idxs)]
         for ch in ch_types_ordered:
             if ch in self.mne.unit_scalings.keys():
                 ch_spinbox = QDoubleSpinBox()
+                ch_spinbox.setMinimumWidth(100)
                 ch_spinbox.setRange(-float("inf"), float("inf"))
-                ch_spinbox.setDecimals(3)
-                ch_spinbox.setValue(self.mne.scalings[ch] * self.mne.unit_scalings[ch])
-                ch_spinbox.setDisabled(True)
-                # ch_spinbox.setReadOnly(True)
-                ch_spinbox.setMinimumWidth(150)
+                ch_spinbox.setDecimals(1)
+                inv_norm = _get_channel_scaling(self, ch)
+                ch_spinbox.setSingleStep(inv_norm * 0.25)
```

Comment:
Have you tried adaptive, might work decently?

https://doc.qt.io/qt-6/qdoublespinbox.html#setStepType
https://doc.qt.io/qt-6/qdoublespinbox.html#stepType-prop